### ISSUE TITLE
IMU example

### ISSUE THREAD

**Issue by rfilkov** [microsoft/Azure-Kinect-Sensor-SDK#667]
**Is your feature request related to a problem? Please describe.**

Currently Sensor SDK provides only raw IMU data. There is no API or example, how to convert the IMU data to device rotation and change in position.

**Describe the solution you'd like**

Please provide SDK API or at least example how to get the device rotation and change in position, according to the raw IMU data coming from device.

**Describe alternatives you've considered**

**Additional context**

**Comment by rfilkov** [comment]
Just published the feature request here: https://feedback.azure.com/forums/920053-azure-kinect-dk/suggestions/38471407-imu-example-or-api I'm not sure though, if this kind of "feature" need to be explicitly requested.

**Comment by prisonerjohn** [comment]
Agreed this would be very useful. Do we need to perform any calibration or correction or is this already handled by the SDK? An example would help answer these questions.
